apps/dashboard/src/routes/root.tsx

```diff
@@ -1,19 +1,39 @@
-import { Outlet } from 'react-router-dom';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { HelmetProvider } from 'react-helmet-async';
-import { withProfiler, ErrorBoundary } from '@sentry/react';
-import { SegmentProvider } from '@/context/segment';
+import { ToastIcon } from '@/components/primitives/sonner';
+import { showToast } from '@/components/primitives/sonner-helpers';
+import { TooltipProvider } from '@/components/primitives/tooltip';
 import { AuthProvider } from '@/context/auth/auth-provider';
 import { ClerkProvider } from '@/context/clerk-provider';
-import { TooltipProvider } from '@/components/primitives/tooltip';
 import { IdentityProvider } from '@/context/identity-provider';
+import { SegmentProvider } from '@/context/segment';
+import { ErrorBoundary, withProfiler } from '@sentry/react';
+import { QueryCache, QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { HelmetProvider } from 'react-helmet-async';
+import { Outlet } from 'react-router-dom';
 
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       refetchOnWindowFocus: false,
     },
   },
+  queryCache: new QueryCache({
+    onError: (_, query) => {
```

Comment:
this will result in toasts shown for not connected bridge and users will see it all the time because we check the status every 10s

Reply:
I know, I'll fix before merging. We'll add a showError meta value and set it to false on those.